packages/web3-eth/types/index.d.ts

```diff
@@ -0,0 +1,268 @@
+    getTransactionCount(address: string): Promise<number>;
+    getTransactionCount(address: string, defaultBlock: number | string): Promise<number>;
+    getTransactionCount(address: string, callback?: (error: Error, count: number) => void): Promise<number>;
+    getTransactionCount(address: string, defaultBlock: number | string, callback?: (error: Error, count: number) => void): Promise<number>;
+    sendTransaction(transaction: Transaction, callback?: (error: Error, hash: string) => void): PromiEvent<TransactionReceipt>;
+    sendSignedTransaction(signedTransactionData: string, callback?: (error: Error, gas: string) => void): PromiEvent<TransactionReceipt>
+    sign(dataToSign: string, address: string | number, callback?: (error: Error, signature: string) => void): Promise<string>;
+    signTransaction(transaction: Transaction, callback?: (error: Error, signedTransaction: SignedTransaction) => void): Promise<SignedTransaction>;
+    signTransaction(transaction: Transaction, address: string): Promise<SignedTransaction>;
+    signTransaction(transaction: Transaction, address: string, callback: (error: Error, signedTransaction: SignedTransaction) => void): Promise<SignedTransaction>;
+    call(transaction: Transaction): Promise<string>;
+    call(transaction: Transaction, defaultBlock?: number | string): Promise<string>;
+    call(transaction: Transaction, callback?: (error: Error, data: string) => void): Promise<string>;
+    call(transaction: Transaction, defaultBlock: number | string, callback: (error: Error, data: string) => void): Promise<string>;
+    estimateGas(transaction: Transaction, callback?: (error: Error, gas: number) => void): Promise<number>;
+    getPastLogs(options: PastLogsOptions, callback?: (error: Error, logs: Log[]) => void): Promise<Log[]>;
+    getWork(callback?: (error: Error, result: string[]) => void): Promise<string[]>;
+    submitWork(data: [string, string, string], callback?: (error: Error, result: boolean) => void): Promise<boolean>;
+}
+
+export interface Methods {
+    property?: string;
+    methods: Method[];
+}
+
+export interface Method {
+    name: string;
+    call: string;
+    params?: number;
+    inputFormatter?: Array<(() => void) | null>;
+    outputFormatter?: () => void;
+}
+
+export interface Syncing {
+    startingBlock: number;
+    currentBlock: number;
+    highestBlock: number;
+    knownStates: number;
+    pulledStates: number;
+}
+
+export interface BlockHeader {
+    number: number
+    hash: string
+    parentHash: string
+    nonce: string
+    sha3Uncles: string
+    logsBloom: string
+    transactionRoot: string
+    stateRoot: string
+    receiptRoot: string
+    miner: string
+    extraData: string
+    gasLimit: number
+    gasUsed: number
+    timestamp: number
+}
+
+export interface Block extends BlockHeader {
+    transactions: Transaction[];
+    size: number
+    difficulty: number
+    totalDifficulty: number
+    uncles: string[];
+}
+
+export interface SignedTransaction {
```

Comment:
SignedTransaction interface is defined twice in this file.

Reply:
oops sorted 👍